modules/gapi/include/opencv2/gapi/s11n.hpp

```diff
@@ -126,53 +212,133 @@ GAPI_EXPORTS std::unique_ptr<IIStream> getInStream(const std::vector<char> &p);
+ * its deserialize() method properly.
+ * @param is IIStream object to deserialize cv::RMat from.
+ * @param r reference to cv::RMat to deserialize into.
+ * @return IIStream object with deserialized and extracted cv::RMat from.
+ * @see deserialize RMat IIStream
+ */
 GAPI_EXPORTS IIStream& operator>> (IIStream& is,       cv::RMat &r);
 
-GAPI_EXPORTS IOStream& operator<< (IOStream& os, const cv::gapi::wip::IStreamSource::Ptr &);
-GAPI_EXPORTS IIStream& operator>> (IIStream& is,       cv::gapi::wip::IStreamSource::Ptr &);
-
-GAPI_EXPORTS IOStream& operator<< (IOStream& os, const cv::detail::VectorRef &);
-GAPI_EXPORTS IIStream& operator>> (IIStream& is,       cv::detail::VectorRef &);
-
-GAPI_EXPORTS IOStream& operator<< (IOStream& os, const cv::detail::OpaqueRef &);
-GAPI_EXPORTS IIStream& operator>> (IIStream& is,       cv::detail::OpaqueRef &);
-
-GAPI_EXPORTS IOStream& operator<< (IOStream& os, const cv::MediaFrame &);
-GAPI_EXPORTS IIStream& operator>> (IIStream& is,       cv::MediaFrame &);
+//! @overload
+GAPI_EXPORTS IOStream& operator<< (IOStream& os, const cv::gapi::wip::IStreamSource::Ptr &issptr);
+//! @overload
+GAPI_EXPORTS IIStream& operator>> (IIStream& is,       cv::gapi::wip::IStreamSource::Ptr &issptr);
+
+//! @overload
+GAPI_EXPORTS IOStream& operator<< (IOStream& os, const cv::detail::VectorRef &vr);
+//! @overload
+GAPI_EXPORTS IIStream& operator>> (IIStream& is,       cv::detail::VectorRef &vr);
+
+//! @overload
+GAPI_EXPORTS IOStream& operator<< (IOStream& os, const cv::detail::OpaqueRef &opr);
+//! @overload
+GAPI_EXPORTS IIStream& operator>> (IIStream& is,       cv::detail::OpaqueRef &opr);
+
+/** @brief This operator is used to serialize cv::MediaFrame into IOStream object.
+ *
+ * It actually serializes the adapter, so the adapter type need to implement
+ * its serialization mechanism.
+ * @note Currently serialization of cv::MediaFrame is not supported.
+ * @param os IOStream object to serialize and store cv::MediaFrame.
+ * @param mf cv::MediaFrame to serialize.
+ * @return IOStream object with serialized cv::MediaFrame inside.
+ * @see serialize MediaFrame IOStream
+ */
+GAPI_EXPORTS IOStream& operator<< (IOStream& os, const cv::MediaFrame &mf);
+/** @brief This operator is used to deserialize cv::MediaFrame from IIStream object.
+ *
+ * It actually deserializes the adapter, so the adapter type need to implement
+ * its deserialization mechanism.
+ * @note Currently deserialization of cv::MediaFrame is not supported.
+ * @param is IIStream object to deserialize cv::MediaFrame from.
+ * @param mf reference to cv::MediaFrame to deserialize into.
+ * @return IIStream object with deserialized and extracted cv::MediaFrame from.
+ * @see deserialize MediaFrame IIStream
+ */
+GAPI_EXPORTS IIStream& operator>> (IIStream& is,       cv::MediaFrame &mf);
 
 // Generic STL types ////////////////////////////////////////////////////////////////
+//! @overload
```

Comment:
Please drop these `overload` things here and below.

Reply:
Dropped